### ISSUE TITLE
[UniProt-GOA] Report on disputed annotation(s) created by GO Reference Genomes

### ISSUE THREAD

**Issue by monicacecilia** [geneontology/go-annotation#1430]
Dear @pgaudet, this comes from JIRA at http://jira.geneontology.org/browse/GO-1251
# 

14/Jun/16 5:03 PM

Hello,

We have identified that one or more annotations created either by you or a curator in your group are the subject of disputes that are yet to be resolved.
Details of the disputed annotation(s) (the entity ID, GO ID, evidence and reference) are listed at the end of this email, along with the name
and email address of the curator who initiated the dispute.

If the issue raised in the dispute has already been addressed, would you please let the initiator know so that they can resolve the dispute.

If you have any concerns regarding the contents of this email, please contact goa@ebi.ac.uk; please DO NOT REPLY to this email - it is sent from an unmonitored account.

Thanks,
The UniProt-GOA team.

```
Date       Dispute ID Initiator               Initiator Email                 Source Entity Type Entity ID   GO ID      Evidence          Reference
2015-04-16 2613       Alexander Deng (User04) alextdeng@gmail.com             REFG   protein     Q30201      GO:0019882 ECO:0000318 (IBA) GO_REF:0000033           
2015-04-16 2614       Alexander Deng (User04) alextdeng@gmail.com             REFG   protein     Q30201      GO:0003823 ECO:0000318 (IBA) GO_REF:0000033           
2015-06-04 2893       Ruth Caroline Lovering  r.lovering@ucl.ac.uk            REFG   protein     P55899      GO:0019882 ECO:0000318 (IBA) GO_REF:0000033           
2015-06-04 2894       Ruth Caroline Lovering  r.lovering@ucl.ac.uk            REFG   protein     P55899      GO:0003823 ECO:0000318 (IBA) GO_REF:0000033           
2015-12-20 3833       Ruth Caroline Lovering  r.lovering@ucl.ac.uk            REFG   protein     P55899      GO:0003823 ECO:0000318 (IBA) GO_REF:0000033           
2015-12-22 3861       Ruth Caroline Lovering  r.lovering@ucl.ac.uk            REFG   protein     P18858      GO:0051103 ECO:0000318 (IBA) GO_REF:0000033           
2016-01-10 3913       Kimberly Van Auken      vanauken@caltech.edu            REFG   protein     Q18317      GO:0043005 ECO:0000318 (IBA) GO_REF:0000033           
2016-01-10 3914       Kimberly Van Auken      vanauken@caltech.edu            REFG   protein     Q18317      GO:0045202 ECO:0000318 (IBA) GO_REF:0000033           
2016-03-15 4233       Kimberly Van Auken      vanauken@caltech.edu            REFG   protein     P28515      GO:0007507 ECO:0000318 (IBA) GO_REF:0000033           
```

**Comment by monicacecilia** [comment]
Dear @pgaudet , 
@RLovering  filed an additional comment about P18858 (included on the table above) back in 22/Dec/15. The issue was not addressed nor closed on JIRA (http://jira.geneontology.org/browse/GO-1074) so I am adding to this open issue for easy access. 

=========
22/Dec/15 7:44 AM
A dispute has been raised over a GO annotation coming from your group. The details are given below.
If you agree with the dispute, please amend or delete the annotation accordingly. If, however, you disagree or need to respond to the disputer, you may contact the disputing curator by replying to this email.

```
SOURCE	ENTITY TYPE	ENTITY ID	GO ID	EVIDENCE	ECO ID	REFERENCE WITH	EXTENSION
REFG	protein	P18858	GO:0051103	IBA	ECO:0000318	GO_REF:0000033 PANTHER:PTN000048794
```

__Dispute ID:__ 3861
__Nature of dispute:__ This dispute concerns some other aspect of the annotation
__Details:__ Hi, please could you explain why this annotation is NOT, PMID:25879709 seem to be working on the assumption that LIG1 is able to repair DNA a statement from this paper: an alternative mechanism of repair occurs called long patch base excision repair (LP-BER), which involves the coordinated activities of Pol polymerase activity and 5 flap incision by flap endonuclease 1 (FEN1) for multiple nucleotide incorporation [22], [23] and [24]. The action of FEN1 results in a nick that is ligated by DNA ligase 1 (LIG1).
Thanks
Ruth

Sent by Protein2GO on behalf of Ruth Caroline Lovering

**Comment by monicacecilia** [comment]
Dear @pgaudet, this comes from an additional report filed under the same subject on 14-Jul (a month after the first report) in JIRA ticket http://jira.geneontology.org/browse/GO-1284

========
We have identified that one or more annotations created either by you or a curator in your group are the subject of disputes that are yet to be resolved.
Details of the disputed annotation(s) (the entity ID, GO ID, evidence and reference) are listed at the end of this email, along with the name
and email address of the curator who initiated the dispute.

If the issue raised in the dispute has already been addressed, would you please let the initiator know so that they can resolve the dispute.

If you have any concerns regarding the contents of this email, please contact goa@ebi.ac.uk; please DO NOT REPLY to this email - it is sent from an unmonitored account.

Thanks,
The UniProt-GOA team.

```
Date       Dispute ID Initiator               Initiator Email                 Source Entity Type Entity ID   GO ID      Evidence          Reference
2015-04-16 2613       Alexander Deng (User04) alextdeng@gmail.com             REFG   protein     Q30201      GO:0019882 ECO:0000318 (IBA) GO_REF:0000033           
2015-04-16 2614       Alexander Deng (User04) alextdeng@gmail.com             REFG   protein     Q30201      GO:0003823 ECO:0000318 (IBA) GO_REF:0000033           
2015-06-04 2893       Ruth Caroline Lovering  r.lovering@ucl.ac.uk            REFG   protein     P55899      GO:0019882 ECO:0000318 (IBA) GO_REF:0000033           
2015-06-04 2894       Ruth Caroline Lovering  r.lovering@ucl.ac.uk            REFG   protein     P55899      GO:0003823 ECO:0000318 (IBA) GO_REF:0000033           
2015-12-20 3833       Ruth Caroline Lovering  r.lovering@ucl.ac.uk            REFG   protein     P55899      GO:0003823 ECO:0000318 (IBA) GO_REF:0000033           
2015-12-22 3861       Ruth Caroline Lovering  r.lovering@ucl.ac.uk            REFG   protein     P18858      GO:0051103 ECO:0000318 (IBA) GO_REF:0000033           
2016-01-10 3913       Kimberly Van Auken      vanauken@caltech.edu            REFG   protein     Q18317      GO:0043005 ECO:0000318 (IBA) GO_REF:0000033           
2016-01-10 3914       Kimberly Van Auken      vanauken@caltech.edu            REFG   protein     Q18317      GO:0045202 ECO:0000318 (IBA) GO_REF:0000033           
2016-03-15 4233       Kimberly Van Auken      vanauken@caltech.edu            REFG   protein     P28515      GO:0007507 ECO:0000318 (IBA) GO_REF:0000033           
2016-06-15 4516       Ruth Caroline Lovering  r.lovering@ucl.ac.uk            REFG   protein     P18858      GO:0051103 ECO:0000318 (IBA) GO_REF:0000033           
2016-06-17 4533       Lionel Breuza           Lionel.Breuza@isb-sib.ch        REFG   protein     P20594      GO:0008074 ECO:0000318 (IBA) GO_REF:0000033           
2016-06-24 4593       Kimberly Van Auken      vanauken@caltech.edu            REFG   protein     G5EG88      GO:0004889 ECO:0000318 (IBA) GO_REF:0000033           
```

**Comment by thomaspd** [comment]
Q30201 was addressed in ticket #1330 
P55899, P18858, Q18317, P28515, P20594 - we checked in amigo and those IBA annotations have been removed
G5EG88 was addressed in ticket #1409
